diff -Naur a/src/api/python/speechd_config/Makefile.am b/src/api/python/speechd_config/Makefile.am
--- a/src/api/python/speechd_config/Makefile.am	2019-05-10 10:40:53.000000000 +0200
+++ b/src/api/python/speechd_config/Makefile.am	2019-10-29 23:53:54.314471175 +0100
@@ -64,14 +64,6 @@
 
 EXTRA_DIST = paths.py.in speechd.desktop.in config.py.in
 
-if HAVE_HELP2MAN
-spd-conf.1: config.py paths.py $(abs_builddir)/__init__.py
-	LC_ALL=C PYTHONPATH=$(builddir)/.. PYTHONDONTWRITEBYTECODE=1 help2man -N --output=$@ $(srcdir)/spd-conf
-
-dist_man1_MANS = \
-	spd-conf.1
-endif
-
 CLEANFILES += $(dist_man1_MANS)
 
 -include $(top_srcdir)/git.mk
